packages/veritone-widgets/src/widgets/EngineSelection/EngineListView/SelectBar/SearchBar.js

```diff
@@ -25,6 +26,24 @@ export default class SearchBar extends React.Component {
     searchQuery: ''
   };
 
+  state = {
+    searchQuery: this.props.searchQuery
+  };
+
+  static getDerivedStateFromProps(nextProps, prevState) {
```

Comment:
this is sketchy pls read https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html and reconsider